frontend/components/Message.svelte

```diff
@@ -15,11 +15,11 @@
     });
 </script>
 
-<p on:click={click} bind:this={element}>
+<p on:click={click} on:keypress={click} bind:this={element}>
```

Comment:
Why is `keypress` handler needed?

Reply:
I got a linter accessibility warning. Because it is a <p> element, it is not clear that you can interact with it. The keypress action allows for disability aids to interact with the element.